tests/platform/broadcom/test_ser.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import random
+import logging
+import pprint
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.broadcom
+]
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DUT_TMP_DIR = os.path.join('tmp', os.path.basename(BASE_DIR))
+
+FILES_DIR = os.path.join(BASE_DIR, 'files')
+SER_INJECTOR_FILE = 'ser_injector.py'
+DUT_WORKING_DIR = '/tmp/'
+
+def disable_ssh_timout(dut):
+    '''
+    @summary disable ssh session on target dut
+    @param dut: Ansible host DUT
+    '''
+    logger.info('Disabling ssh time out on dut: %s' % dut.hostname)
+    dut.command("sudo sed -i 's/^ClientAliveInterval/#&/' /etc/ssh/sshd_config")
+    dut.command("sudo sed -i 's/^ClientAliveCountMax/#&/' /etc/ssh/sshd_config")
+
+    dut.command("sudo systemctl restart ssh")
+    time.sleep(5)
+
+def enable_ssh_timout(dut):
+    '''
+    @summary: enable ssh session on target dut
+    @param dut: Ansible host DUT
+    '''
+    logger.info('Enabling ssh time out on dut: %s' % dut.hostname)
+    dut.command("sudo sed -i '/^#ClientAliveInterval/s/^#//' /etc/ssh/sshd_config")
+    dut.command("sudo sed -i '/^#ClientAliveCountMax/s/^#//' /etc/ssh/sshd_config")
+
+    dut.command("sudo systemctl restart ssh")
+    time.sleep(5)
+
+@pytest.fixture(scope='module')
+def setup(testbed_devices):
```

Comment:
we should merge this in the original testbed_devices, with parameterized, to disable/enable ssh timeout configuration.

https://docs.pytest.org/en/latest/fixture.html#fixture-parametrize

Reply:
Done